### PR TITLE
dmd 2.068.0-b2 and HEAD

### PR DESCRIPTION
**Pull request by John-Colvin**

### CODE REVIEW

**Comment by John-Colvin** [comment]
Is sha1 ok for now or should I change them all to sha256?

**Comment by dunn** [comment]
All of them, please!

**Comment by John-Colvin** [comment]
done

**Comment by John-Colvin** [comment]
Can't reproduce these errors locally. I'll try adding some commits here to dump some more information from the autotester

**Comment by John-Colvin** [comment]
Can anyone give me a hand with this? From the command line output in the autotester it looks like ```@@HOMEBREW_CELLAR@@``` ended in the dmd.conf file, which I can't see any reason for.

**Comment by John-Colvin** [comment]
Also I can't see any of the extra output fromhttps://github.com/John-Colvin/homebrew/commit/ba7cd04bf19751755fe03fbb4528939e1c28b32d in the autotester logs, works fine locally.

**Comment by xu-cheng** [comment]
Should fix now. Please upgrade PR to remove the debug code.

**Comment by John-Colvin** [comment]
Done

File: Library/Formula/dmd.rb [hunk 1]
```diff
@@ -12,25 +28,47 @@ class Dmd < Formula
...
+    resource "tools" do
+      url "https://github.com/D-Programming-Language/tools.git"
+    end
   end
 
   def install
     make_args = ["INSTALL_DIR=#{prefix}", "MODEL=#{Hardware::CPU.bits}", "-f", "posix.mak"]
 
-    system "make", "SYSCONFDIR=#{etc}", "TARGET_CPU=X86", "RELEASE=1", *make_args
+    system "make", "SYSCONFDIR=#{etc}", "TARGET_CPU=X86", "AUTO_BOOTSTRAP=1", "RELEASE=1", *make_args
```
Comment (dunn) [inline Library/Formula/dmd.rb]: Does `AUTO_BOOTSTRAP=1` need to be used for the stable build, as well as the HEAD one?

Comment (John-Colvin) [inline Library/Formula/dmd.rb]: It's a no-op for 2.067.1 and will be a necessity for 2.068.0, whether stable or not.

**Comment by dunn** [comment]
Merged in b4f070f3a5ca95b0f89c6f0745273ddc32485951, thanks for your patience on this!

**Pull request closed without merging**